lib/MetaCPAN/Script/Release.pm

```diff
@@ -237,6 +237,15 @@ sub import_archive {
             = [ @{ $associated_pod{$documentation} || [] }, $_ ];
     }
 
+# check for release deprecation in abstract of release or has x_deprecated in meta
+    my $deprecated = (
+               $meta->{x_deprecated}
+            or $document->has_abstract
+            and $document->abstract =~ /DEPRECI?ATED/
+    ) ? 1 : 0;
+
+    $document->_set_deprecated($deprecated);
```

Comment:
First off, thanks so much for doing this.  I have wanted to see something like this for a long time.  My only real issue with the logic is that it seems to blur the lines between a release which is deprecated and a release which contains only *some* deprecated modules.  See https://metacpan.org/pod/release/ETHER/Dist-Zilla-Plugin-Deprecated-0.007/lib/Dist/Zilla/Plugin/Deprecated.pm#DESCRIPTION

Basically, checking for presence of the key will not always be quite enough.  If a release has 10 modules and one of them has been deprecated using `x_deprecated` key, then it looks like the entire release will be flagged.

Reply:
OK, that clarifies the issue I ran into with this :)
I'll try to fix the logic.

Reply:
This one is indicating a release deprecation, i.e. the x_deprecated is in the top level of meta.
A single deprecated file does not set this flag, in that case we will have the x_deprecated key set under `meta->provides->file->x_deprecated` so that logic is just missing for the file later on - will add that.

Reply:
added the mentioned check on file level